gramlib/grammar.ml

```diff
@@ -1652,6 +1654,9 @@ module Parsable = struct
 
   let comments p = L.State.get_comments !(p.lexer_state)
 
+  let loc t = LStream.current_loc t.pa_tok_strm
+  let consume { pa_tok_strm } len =
+    for _i = 0 to len-1 do LStream.junk pa_tok_strm done
```

Comment:
I would rather add `njunk` to LStream (it's already in Stream)

Reply:
Done, thanks. I've kept the `consume` name here, but maybe we should change it too? I dunno.